pkg/controller/node/nodecontroller.go

```diff
@@ -111,7 +125,8 @@ func NewNodeController(
 	nodeStartupGracePeriod time.Duration,
 	nodeMonitorPeriod time.Duration,
 	clusterCIDR *net.IPNet,
-	allocateNodeCIDRs bool) *NodeController {
+	allocateNodeCIDRs bool,
+	resyncPeriod controller.ResyncPeriodFunc) *NodeController {
```

Comment:
why did you make this a parameter if you only ever call this function with the same value (controller.NoResyncPeriodFunc) ?